apps/total-typescript/src/certificate/workshop-certificate.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import {Module} from '@skillrecordings/skill-lesson/schemas/module'
 import {LockClosedIcon} from '@heroicons/react/solid'
 import Image from 'next/image'
-import {trpc} from 'trpc/trpc.client'
+import {trpc} from '@/trpc/trpc.client'
```

Comment:
next added a line to the tsconfig that called for files inside of the `.next` dir and TS hated it, so I had to bail on our config since it decalered a `rootDir` which is what made this possible. This use of an alias is more standard.